packages/flutter/pubspec.yaml

```
@@ -1,16 +1,16 @@
 name: parse_server_sdk_flutter
-description: The Flutter SDK for Parse Platform (https://parseplatform.org)
+description: The Flutter SDK to connect to Parse Server. Build your apps faster with Parse Platform, the complete application stack.
 version: 5.0.0
```

We could publish a new release to get a better score right away. Alternatively we'd have to wait for the next PR that triggers a release to get a better score. If we publish a release now, we'd also need a changelog entry.

```suggestion
version: 5.1.0
```

---

In Dart's PR, I said that I think the refactor is better, but if it is very necessary, a new version can be published
@Nidal-Bakir 
What is your opinion ?

---

@mtrezza 
Is this PR ready to be merged?

---

So just like with https://github.com/parse-community/Parse-SDK-Flutter/pull/883 we won't publish a new release with this PR, so it's just a `refactor` without version increment.